centrifuge-app/src/components/PoolCard/index.tsx

```diff
@@ -76,42 +110,82 @@ export function PoolCard({
   metaData,
   createdAt,
 }: PoolCardProps) {
+  const theme = useTheme()
   const isOneTranche = tranches && tranches?.length === 1
-  const renderText = (text: string) => (
-    <Text fontWeight={500} as="h2" variant="body1">
-      {text}
-    </Text>
-  )
+  const isTinlakePool =
+    poolId === '0x53b2d22d07E069a3b132BfeaaD275b10273d381E' ||
+    poolId === '0x90040F96aB8f291b6d43A8972806e977631aFFdE' ||
+    poolId === '0x55d86d51Ac3bcAB7ab7d2124931FbA106c8b60c7'
 
-  const tranchesData = tranches
-    ?.map((tranche) => {
-      const words = tranche.currency.name.trim().split(' ')
-      const metadata = metaData?.tranches[tranche.id] ?? null
-      const trancheName = words[words.length - 1]
-      const investmentBalance = new CurrencyBalance(
-        metadata?.minInitialInvestment ?? 0,
-        tranche.currency.decimals
-      ).toDecimal()
-
-      const daysSinceCreation = createdAt ? daysBetween(createdAt, new Date()) : 0
-
-      function calculateApy() {
-        if (poolId === '4139607887') return formatPercentage(5, true, {}, 1)
-        if (poolId === '1655476167') return formatPercentage(15, true, {}, 1)
-        if (daysSinceCreation > 30 && tranche.yield30DaysAnnualized)
-          return formatPercentage(tranche.yield30DaysAnnualized, true, {}, 1)
-        if (tranche.interestRatePerSec) return formatPercentage(tranche.interestRatePerSec.toAprPercent(), true, {}, 1)
-        return '-'
-      }
-
-      return {
-        name: trancheName,
-        apr: calculateApy(),
-        minInvestment:
-          metadata && metadata.minInitialInvestment ? `$${formatBalanceAbbreviated(investmentBalance, '', 0)}` : '-',
-      }
-    })
-    .reverse()
+  const tinlakeObjKey = () => {
+    if (name?.includes('Silver')) return 'silver'
```

Comment:
```suggestion
    if (name?.includes('New Silver 3')) return 'NS3'
```
I think you need to be more specific here since we have two other New Silver pools (in the archive)